igor-monitor-travis/src/main/java/com/netflix/spinnaker/igor/travis/config/TravisConfig.java

```diff
@@ -83,17 +82,22 @@ public Map<String, TravisService> travisMasters(
                               igorConfigurationProperties.getClient().getTimeout(),
                               objectMapper);
 
-                      boolean useLegacyLogFetching = false;
```

Comment:
For the record, it looks like this came from https://github.com/spinnaker/igor/pull/449.

Reply:
Yes, correct. We had high hopes for this feature 😢